javascript/javascript2/week3/TheFastestPresserInThisRealm/TheFastestPresserInThisRealm.js

```diff
@@ -0,0 +1,108 @@
+let time = 0,
+    pressL = 0,
+    pressS = 0;
+
+const buttonStart = document.querySelector('.start'),
+      buttonRestart = document.querySelector('.restart'),
+      input = document.querySelector('input'),
+      scoreS = document.querySelector('.scoreS'),
+      scoreL = document.querySelector('.scoreL'),
+      wrap = document.querySelector('.wrapper'),
+      konfetti = document.querySelector('#my-canvas'),
+      clock = document.createElement('div'),
+      message = wrap.appendChild(clock);
+
+clock.classList.add('timer');
+
+function buttonsFoGame(e) {
+    if (e.code === 'KeyS') {
+        pressS += 1;    
+    } else if (e.code === 'KeyL') {
+        pressL += 1;    
+    } else {
+        alert('You pressed wrong button');
+    }
+    scoreS.innerHTML = pressS;
+    scoreL.innerHTML = pressL;
+}
+
+
+function startGame(){
+    buttonStart.addEventListener('click', () => {
```

Comment:
good job. Dmytrii. 
the other option could be here is  to do startGame function and then call it in eventListner from outside . something like 
buttonStart.addEventListener('click', startGame);
function startGame() {

  time = +input.value;
  if (input.value == '') {
    alert(`You don't set time of game`);
    return;
  } else if (!input.value.match(/[0-9]/)) {
    alert(`You have to input number`);
    return;
  }
  document.addEventListener('keyup', buttonsFoGame);
  startTimer();

  setTimeout(function () {
    document.removeEventListener('keyup', buttonsFoGame);
  }, time * 1000);


}